From 1493b0b35ab422e947857f3fdfc55d32379180b3 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 3 Jan 2019 09:27:07 -0800
Subject: [PATCH] TRITON-760 imgapi-external-manta-setup and imgapi-manta-setup
 were broken by sdc-clients 11.x update in TRITON-53 (missed some SAPI client
 usage)

---
 bin/imgapi-manta-setup | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/bin/imgapi-manta-setup b/bin/imgapi-manta-setup
index c89f8db..0166483 100755
--- a/bin/imgapi-manta-setup
+++ b/bin/imgapi-manta-setup
@@ -6,7 +6,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 /*
@@ -311,6 +311,7 @@ function generateKey(keyName, callback) {
 function getSdcApp(config, callback) {
     var sapi = new sdcClients.SAPI({
         url: 'http://' + config.sapi_service,
+        version: '~2',
         log: log,
         agent: false
     });
@@ -345,6 +346,7 @@ function ensureImgapiSapiServiceSetup(args, callback) {
         args.sdcApp.metadata.datacenter_name);
     var sapi = new sdcClients.SAPI({
         url: 'http://' + args.config.sapi_service,
+        version: '~2',
         log: log,
         agent: false
     });
-- 
2.21.0

